sdkproject/Assets/Mapbox/Examples/Scripts/DynamicZoomCameraMovement.cs

```diff
@@ -60,7 +64,7 @@ private void LateUpdate()
 			// zoom
 			var y = Input.GetAxis("Mouse ScrollWheel") * _zoomSpeed;
 			//avoid unnecessary translation
-			if (0 != y)
+			if (Mathf.Abs(y) > 0.0f)
```

Comment:
Just curious, what's the advantage of using `Mathf.Abs`?

Reply:
Removes the compiler warning and to avoid precision errors.